gym/wrappers/monitoring.py

```diff
@@ -0,0 +1,41 @@
+from gym import monitoring
+from gym import Wrapper
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+def Monitor(directory, video_callable=None, force=False, resume=False,
```

Comment:
if this is in reference to writing `Monitor(directory)(env)` I agree. It's unexpected